vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -23602,3 +23602,106 @@ THE UCUM TABLE (IN ALL FORMATS), UCUM DEFINITIONS, AND SPECIFICATION ARE PROVIDE
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "All Units Ontology Version 2.1.16" ;
 .
+
+unit:DEG_C_GROWING_CEREAL-DAY
+    a qudt:Unit ;
+    rdfs:label "Growing Degree Days (Cereals)" ;
+    qudt:conversionMultiplier "86400"^^xsd:double ;
+    qudt:conversionOffset "0.0"^^xsd:double ;
+    qudt:hasDimensionVector qkdv:A0E0L0I0M0H1T1D0 ;
+    qudt:plainTextDescription "The sum of excess temperature over 5.5°C, where the temperature is the mean of the minimum and maximum atmospheric temperature in a day. This measure is appropriate for most cereal crops." ;
+    qudt:hasQuantityKind quantityKind:GrowingDegreeDay_Cereal ;
+    rdfs:isDefinedBy <https://data.agrimetrics.co.uk/ontologies/qudt-extension> .
+
+unit:M2-PER-M2
+    a  qudt:Unit;
+    rdfs:label "square metre per square metre" ;
+    qudt:hasDimensionVector qkdv:A0E0L2I0M0H0T0D0 ;
```

Comment:
This should be dimensionless, i.e. qkdv:A0E0L0I0M0H0T0D1. To distinguish this dimensionless ratio from others, I recommend you also include:
  qudt:qkdvDenominator qkdv:A0E0L2I0M0H0T0D0 ;
  qudt:qkdvNumerator qkdv:A0E0L2I0M0H0T0D0 ;